src/components/Private/Settings.tsx

```diff
@@ -0,0 +1,197 @@
+import Container from '@material-ui/core/Container';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Link from '@material-ui/core/Link';
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper';
+import PlaceholderProfileImg from '../../assets/img/kitten_placeholder.jpg'; 
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+const styles = (theme: Theme) => createStyles({
+  paper: {
+    margin: theme.spacing(2, 0, 0, 0),
+    padding: theme.spacing(3, 4, 3, 2),
+  },
+  wrapper: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center'
+  },
+  avatar: {
+    margin: theme.spacing(1),
+    backgroundColor: theme.palette.secondary.main,
+    width: '125px',
+    height: '125px'
+  },
+  profilePicLink: {
+    color: "#808080"
+  },
+  statistics: {
+    margin: theme.spacing(2, 0, 0, 0)
+  },
+  buttons: {
+    margin: theme.spacing(2, 0, 0, 0)
+  }
+});
+
+
+type State = { contactOpen: boolean, passwordOpen : boolean };
+interface Props extends WithStyles<typeof styles>{ }
+
+class Settings extends Component<Props,State> {
+  constructor(props: any) {
+    super(props)
+    this.state = {
+        contactOpen : false,
+        passwordOpen : false,
+    }
+    this.handleContactOpen = this.handleContactOpen.bind(this)
+    this.handleContactClose = this.handleContactClose.bind(this)
+    this.handlePasswordOpen = this.handlePasswordOpen.bind(this)
+    this.handlePasswordClose = this.handlePasswordClose.bind(this)
+    
+  }
+
+  handleContactOpen = () => {
+    this.setState({
+        contactOpen : true
+    })
+  };
+
+  handleContactClose = () => {
+    this.setState({
+        contactOpen : false
+    })
+  };
+
+  handlePasswordOpen = () => {
+    this.setState({
+        passwordOpen : true
+    })
+  };
+
+  handlePasswordClose = () => {
+    this.setState({
+        passwordOpen : false
+    })
+  };
+
+  render() {
+    const { classes } = this.props;
+    return (
+      <Container component="main" maxWidth="sm">
+        <CssBaseline />
+        <Paper elevation={3} className={classes.paper}>
+          <div className={classes.wrapper}>
+            <Avatar alt="profile photo" src={PlaceholderProfileImg} className={classes.avatar} />
+            <Typography component="h1" variant="h4">
+              John Doe
+            </Typography>
+            <Typography>
+              <Link href="#" className={classes.profilePicLink}>Change profile picture</Link>
+            </Typography>
+            <Typography className={classes.statistics}>
+              Email: JDoe@ukg.com
+            </Typography>
+            <Typography>
+              Phone Number: (401)-867-5309
+            </Typography>
+
```

Comment:
Add in a part for user id or username
```suggestion
<Typography>
    User ID: 24610
</Typography>
```